spring-cloud-azure-context/src/main/java/com/microsoft/azure/spring/cloud/context/core/config/AzureProperties.java

```diff
@@ -28,6 +29,8 @@
 
     private boolean autoCreateResources = false;
 
+    private boolean secureTransfer = true;
```

Comment:
Could we move this property into storage properties? It seems it only applies to storage account.

Reply:
Good point. Pushin the changes right now.